doc/man/CMakeLists.txt

```diff
@@ -25,7 +25,7 @@ macro(add_manpage pod)
   set(target ${CMAKE_CURRENT_BINARY_DIR}/${manpage})
   add_custom_command(
     OUTPUT ${target}
-    COMMAND ${CMAKE_SOURCE_DIR}/tools/makeman.sh ${source} ${CMAKE_CURRENT_SOURCE_DIR}/../../AUTHORS ${target}
+    COMMAND bash ${CMAKE_SOURCE_DIR}/tools/makeman.sh ${source} ${CMAKE_CURRENT_SOURCE_DIR}/../../AUTHORS ${target}
```

Comment:
Why bash and not sh? Shebang says sh, if it really requires bash, then it should be changed too.

Reply:
Right you are! changed to sh :)